sklearn/titanic/README.md

```diff
@@ -0,0 +1,99 @@
+## Introduction
+
+This sample code illustrates how to start a GCE instance and train a model using scikit-learn on the instance. We will be using the [Titanic dataset](https://www.kaggle.com/c/titanic) from [Kaggle](https://www.kaggle.com/). The goal is to train a classifier to predict whether a given passenger on Titanic survived or not.
+
+## Prerequisites
+
+You need to download [train.csv](https://www.kaggle.com/c/titanic/download/train.csv) from [Kaggle](https://www.kaggle.com/).
```

Comment:
If you are not logged into Kaggle when you first click on the `train.csv` link, it prompts you to log in and then takes you to  your Kaggle main page instead of the Titanic dataset page - this could be slightly disorienting. Should we prepare users for this kind of disorientation here?